packages/office-addin-dev-certs/src/defaults.ts

```diff
@@ -9,6 +9,8 @@ export const certificateDirectoryName = ".office-addin-dev-certs";
 export const certificateDirectory =  path.join(os.homedir(), certificateDirectoryName);
 export const caCertificateFileName = "ca.crt";
 export const caCertificatePath = path.join(certificateDirectory, caCertificateFileName);
+export const localhostCaCertificateFileName = "ca.crt";

```

Comment:
`localhostCaCertificateFileName` and `localhostCaCertificatePath` seems to duplicate the same thing that is already provided by `caCertificateFileName` and `caCertificatePath`.

Reply:
I guess I don't need to include localhostCaCertificateFileName and localhostCaCertificatePath. I could've sworn I was getting a test failure without this change but must have been some other reason.  Removing them